paddle/fluid/framework/operator.cc

```diff
@@ -2369,7 +2371,10 @@ Scope* OperatorWithKernel::PrepareData(
   };
 
   if (run_phi_kernel_) {
-    const auto& input_names = kernel_signature_->input_names;
+    std::vector<std::string> input_names;
```

Comment:
Because the member of input_names is `char*`, and we need to ensure a input name whether in the `input_names `, the `find` operation need to trans input_names from `vector<char*>` to `vector<string>`

Reply:
we can use the recored extra input names to avoid create this vector

Reply:
done, thx